Unexpected keyword argument in generated unit tests for streaming methods

Affected APIs:
```
//google/cloud/bigquery/storage/v1/storage_py_gapic_test
//google/cloud/bigquery/storage/v1beta2/storage_py_gapic_test
//google/devtools/build/v1/build_py_gapic_test
```

Error output:
```
_______________________ test_append_rows_flattened_error _______________________

    def test_append_rows_flattened_error():
        client = BigQueryWriteClient(
            credentials=ga_credentials.AnonymousCredentials(),
        )
    
        # Attempting to call a method with both a request object and flattened
        # fields is an error.
        with pytest.raises(ValueError):
>           client.append_rows(
                storage.AppendRowsRequest(),
                write_stream='write_stream_value',
            )
E           TypeError: BigQueryWriteClient.append_rows() got an unexpected keyword argument 'write_stream'
```

--- a/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
+++ b/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
@@ -363,7 +363,7 @@ def test_{{ method_name }}_from_dict_foreign():
 
 {% endif %}
 
-{% if method.flattened_fields %}
+{% if method.flattened_fields and not method.client_streaming %}
 def test_{{ method_name }}_flattened():
     client = {{ service.client_name }}(
         credentials=ga_credentials.AnonymousCredentials(),
@@ -1224,7 +1224,7 @@ def test_{{ method_name }}_rest_bad_request(transport: str = 'rest', request_typ
         {% endif %}
 
 
-{% if method.flattened_fields %}
+{% if method.flattened_fields and not method.client_streaming %}
 def test_{{ method_name }}_rest_flattened():
     client = {{ service.client_name }}(
         credentials=ga_credentials.AnonymousCredentials(),

